src/ggrc/converters/handlers/custom_attribute.py

```
@@ -80,13 +85,17 @@ def insert_object(self):
     self.dry_run = True
 
   def get_ca_definition(self):
-    for definition in self.row_converter.object_class\
-            .get_custom_attribute_definitions():
-      if definition.title == self.display_name:
-        return definition
-    return None
+    """Get custom attribute definition."""
+    cad = models.CustomAttributeDefinition
+    definition_type = self.row_converter.obj._inflector.table_singular
+    return cad.eager_query().filter(and_(
+        cad.definition_type == definition_type,
+        cad.definition_id.is_(None),
+        cad.title == self.display_name,
+    )).first()
```

Also do we do any error handling if self.display_name is not an actual custom attribute? `first()` will just throw an exception in this case, right. Will the exception be silently ignored or will we notify the user about it?